automated install/basic-install.sh

```diff
@@ -52,16 +52,20 @@ lighttpdConfig=/etc/lighttpd/lighttpd.conf
 # This is a file used for the colorized output
 coltable=/opt/pihole/COL_TABLE
 
+# Root of the web server
+webroot="/var/www/html"
+
 # We store several other directories and
 webInterfaceGitUrl="https://github.com/pi-hole/AdminLTE.git"
-webInterfaceDir="/var/www/html/admin"
+webInterfaceDir="${webroot}/admin"
```

Comment:
Whilst I'm looking at this, this could probably be renamed to `PI_HOLE_WEBINTERFACE_DIR` or `PI_HOLE_ADMIN_DIR` or something. 

Keep in mind that if this variable name is changed, it must also be updated in the [checkout script](https://github.com/pi-hole/pi-hole/blob/995ee41d6bc3a405a0402745b24140ca08c148f3/advanced/Scripts/piholeCheckout.sh)

Any thoughts @pi-hole/core-approvers ?

Reply:
Feel free to open a PR for this ;-)

Reply:
We should not prefix with `PI_HOLE`, as then everything would require an extra prefix. There is no issue with not having the prefix.